peer.go

```diff
@@ -2527,6 +2534,11 @@ func (p *peer) sendInitMsg() error {
 // resendChanSyncMsg will attempt to find a channel sync message for the closed
 // channel and resend it to our peer.
 func (p *peer) resendChanSyncMsg(cid lnwire.ChannelID) error {
+	// If we already re-sent the mssage for this channel, we won't do it again.
+	if _, ok := p.resentChanSyncMsg[cid]; ok {
```

Comment:
We don't appear to actually populate the map in this diff as is.

Reply:
Wow, that's embarrassing ... 🤦‍♂ fixed